File: Lenguajes_Programacion/C#/1_Sintaxis_C#/app/z_Modulos/lenguaje.cs
class Lenguaje
{
    private string nombre;
    private int anio;
    public Lenguaje(string nombre, int anio)
    {
        this.nombre = nombre;
        this.anio = anio;
    }

    public void descripcion()
    {
        Console.WriteLine("{0} fue creado en {1}", this.nombre, this.anio);
    }

}

File: 02_Lenguajes_Programacion/1_Java/1_sintaxis_java/modulos/lenguaje.java
package modulo;


public class Lenguaje {
    String nombre;
    Integer anio;

    public Lenguaje() {}

    public Lenguaje(String nombre, Integer anio) {
        this.nombre = nombre;
        this.anio = anio;
    }

    public void descripcion() {
        System.out.println(this.nombre + " fue creado en " + this.anio);
    }
}

// Para compilar este modulo se debe ejecutar usando el nombre de la clase así: "javac Lenguaje.java"